packages/basis/src/Basis.ts

```diff
@@ -89,7 +89,7 @@ export const BASIS_FORMATS_ALPHA: { [id: number]: boolean } = {
  * Binding to C++ {@code BasisFile} wrapper class.
  *
  * @see https://github.com/BinomialLLC/basis_universal/blob/master/webgl/transcoder/basis_wrappers.cpp
- * @ignore
```

Comment:
@ShukantPal is this a webdoc regression? Seems like exporting a class doesn't exclude it from showing up. I had to use `@private` here, but I would expect `@ignore` to work too.

Reply:
Interesting - it might be related to using `declare`, but not sure.

Reply:
I also saw this behavior working on resource-loader PR. AsyncQueue.ts file, only `@private` worked not `@ignore` and this didn't have `declare`.

Reply:
@ShukantPal https://github.com/webdoc-labs/webdoc/issues/132